conda-store-server/Dockerfile

```diff
@@ -1,7 +1,10 @@
-FROM --platform=linux/amd64 condaforge/mambaforge:23.3.1-1
+FROM --platform=linux/amd64 condaforge/mambaforge:23.3.1-1 as base
 
 LABEL org.opencontainers.image.authors="conda-store development team"
 
+ENV PATH=/opt/conda/condabin:/opt/conda/envs/conda-store-server/bin:/opt/conda/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:${PATH}
+ENV TZ=America/New_York
```

Comment:
Is this the same timezone we use for conda-store general config? I thought we were using UTC

Reply:
This is what it had. I think UTC would make more sense personally, but I didn't want to change it without having context around why it was set to that. Would you like me to change it?